src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -468,6 +481,75 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         driver.swipe(from.getX(), from.getY(), to.getX(), to.getY(), SWIPE_DURATION_MILLIS);
         return this;
     }
+  
+    private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
+        Dimension screen = driver.manage().window().getSize();
+
+        Point from;
+        if(corner != null) {
+            int x = 0;
+            int y = 0;
+            switch(corner) {
+                case TOP_LEFT:
+                    x = screen.getWidth() - 90;
```

Comment:
the left border is not always `width - 90`. If the screen has a width of 500 pixels then you get a y of 410. You want something much close to 0 (the left border).
see here for some pixel values http://www.javascriptkit.com/dhtmltutors/cssmediaqueries2.shtml

Reply:
I'm fairly certain this is in percent values? I could be wrong but if you look at the performCenterSwipe, it uses 50 as the x and y start value which to me seems to mean that it is using percent values. @jossjacobo is this true?

Reply:
It's calling `AppiumDriver::swipe()` which is an implementation of https://static.javadoc.io/io.appium/java-client/3.3.0/io/appium/java_client/TouchShortcuts.html#swipe(int,%20int,%20int,%20int,%20int)
It should be coordinates.

Reply:
Updated with latest commit :)